starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,56 @@
 <!DOCTYPE html>
```

Comment:
Muy bien por esta parte, espero que hayas aprovechado la funcionalidad built-in de VSCode, emmet. Si esto te suena a chino, ven y te lo explico